images/build/Dockerfiles/cli.Dockerfile

```diff
@@ -58,6 +58,7 @@ RUN if [ "${DEBIAN_FLAVOR}" = "buster" ]; then \
 
```

Comment:
I think you would also need [ENV DEBIAN_FRONTEND=noninteractive](https://github.com/microsoft/Oryx/pull/2336/files#diff-0d4b900c8211fac42c59c87aea1f3e18c0757130bedb141ca476b4814f818db1R24) for both Dockerfiles